core/src/main/java/io/parsingdata/metal/data/selection/ByName.java

```diff
@@ -34,7 +33,7 @@ private ByName() {}
      * @return The first value (bottom-up) with the provided name in the provided graph
      */
     public static ParseValue getValue(final ParseGraph graph, final String name) {
-        return ByPredicate.getAllValues(ImmutableList.create(graph), new ImmutableList<>(), (value) -> value.matches(name), 1).computeResult().head;
+        return ByPredicate.getAllValues(graph, (value) -> value.matches(name), 1).head;
```

Comment:
Before there was a nice public and static method exposing the `limit` argument, the only way to use it was by directly calling the internal (package private) method. Essentially what I forgot to do with the `NameRef` constructor above :)